From 1a3938e8c133573f35abe58edd42204575aeffeb Mon Sep 17 00:00:00 2001
From: tastytea <tastytea@tastytea.de>
Date: Fri, 8 Sep 2023 20:01:03 +0200
Subject: [PATCH] skip some tests

reasons:
  - needs to be in a git repo
  - needs special test dependencies
  - requires network access
---
 modules/client_test.go                                        | 4 ++++
 .../resource_factories/create/create_integration_test.go      | 1 +
 2 files changed, 5 insertions(+)

diff --git a/modules/client_test.go b/modules/client_test.go
index ea910580f..b957ba984 100644
--- a/modules/client_test.go
+++ b/modules/client_test.go
@@ -68,6 +68,7 @@ github.com/gohugoio/hugoTestModules1_darwin/modh2_2@v1.4.0 github.com/gohugoio/h
 	}
 
 	c.Run("All", func(c *qt.C) {
+		c.Skip("Test dependencies not available")
 		client, clean := newClient(c, func(cfg *ClientConfig) {
 			cfg.ModuleConfig = DefaultModuleConfig
 		}, defaultImport)
@@ -107,6 +108,7 @@ project github.com/gohugoio/hugoTestModules1_darwin/modh2_2_2@v1.3.0+vendor
 	})
 
 	c.Run("IgnoreVendor", func(c *qt.C) {
+		c.Skip("Test dependencies not available")
 		client, clean := newClient(
 			c, func(cfg *ClientConfig) {
 				cfg.ModuleConfig = DefaultModuleConfig
@@ -125,6 +127,7 @@ project github.com/gohugoio/hugoTestModules1_darwin/modh2_2_2@v1.3.0+vendor
 	})
 
 	c.Run("NoVendor", func(c *qt.C) {
+		c.Skip("Test dependencies not available")
 		mcfg := DefaultModuleConfig
 		mcfg.NoVendor = "**"
 		client, clean := newClient(
@@ -144,6 +147,7 @@ project github.com/gohugoio/hugoTestModules1_darwin/modh2_2_2@v1.3.0+vendor
 	})
 
 	c.Run("VendorClosest", func(c *qt.C) {
+		c.Skip("Test dependencies not available")
 		mcfg := DefaultModuleConfig
 		mcfg.VendorClosest = true
 
diff --git a/resources/resource_factories/create/create_integration_test.go b/resources/resource_factories/create/create_integration_test.go
index 61bc17adb..13cd6f33f 100644
--- a/resources/resource_factories/create/create_integration_test.go
+++ b/resources/resource_factories/create/create_integration_test.go
@@ -25,6 +25,7 @@ import (
 )
 
 func TestGetRemoteHead(t *testing.T) {
+	t.Skip("Requires network access")
 	files := `
 -- config.toml --
 [security]
-- 
2.43.0

